### ISSUE TITLE
LitEmoji exception with HTML form field

### ISSUE THREAD

**Issue by martinleveille** [verbb/formie#1381]
### Describe the bug

When you create a new form with the HTML field, you're unable to save the form.

When you try to edit an existing form that have an HTML field, you're unable to edit the form.

### Steps to reproduce

1. Create a new form
2. Add an HTML form field
3. Save the form

### Form settings

- Multi-page form: No
- Submission Method: Page Reload
- Client-side Validation: No
- Custom Form Templates: No

### Craft CMS version

4.4.6.1

### Plugin version

2.0.26

### Multi-site?

Yes

### Additional context

```
TypeError: LitEmoji\LitEmoji::shortcodeToUnicode(): Argument #1 ($content) must be of type string, null given, called in /var/www/html/vendor/verbb/formie/src/fields/formfields/Html.php on line 78 and defined in /var/www/html/vendor/elvanto/litemoji/src/LitEmoji.php:62
```

Stack trace:

```
#0 /var/www/html/vendor/verbb/formie/src/fields/formfields/Html.php(78): LitEmoji\LitEmoji::shortcodeToUnicode(NULL)
#1 /var/www/html/vendor/verbb/formie/src/services/Fields.php(541): verbb\formie\fields\formfields\Html->getSavedFieldConfig()
#2 /var/www/html/vendor/verbb/formie/src/services/Fields.php(932): verbb\formie\services\Fields->getSavedFieldConfig(Object(verbb\formie\fields\formfields\Html))
#3 /var/www/html/vendor/verbb/formie/src/elements/Form.php(590): verbb\formie\services\Fields->getRowConfig(Array)
#4 /var/www/html/vendor/verbb/formie/src/controllers/FormsController.php(491): verbb\formie\elements\Form->getFormConfig()
#5 /var/www/html/vendor/verbb/formie/src/controllers/FormsController.php(106): verbb\formie\controllers\FormsController->_prepareVariableArray(Array)
#6 [internal function]: verbb\formie\controllers\FormsController->actionEdit(1175, NULL, NULL)
#7 /var/www/html/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array)
#8 /var/www/html/vendor/yiisoft/yii2/base/Controller.php(178): yii\base\InlineAction->runWithParams(Array)
#9 /var/www/html/vendor/yiisoft/yii2/base/Module.php(552): yii\base\Controller->runAction('edit', Array)
#10 /var/www/html/vendor/craftcms/cms/src/web/Application.php(302): yii\base\Module->runAction('formie/forms/ed...', Array)
#11 /var/www/html/vendor/yiisoft/yii2/web/Application.php(103): craft\web\Application->runAction('formie/forms/ed...', Array)
#12 /var/www/html/vendor/craftcms/cms/src/web/Application.php(287): yii\web\Application->handleRequest(Object(craft\web\Request))
#13 /var/www/html/vendor/yiisoft/yii2/base/Application.php(384): craft\web\Application->handleRequest(Object(craft\web\Request))
#14 /var/www/html/web/index.php(12): yii\base\Application->run()
#15 {main}
```

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, run `composer require verbb/formie:"dev-craft-4 as 2.0.26"`

**Comment by engram-design** [comment]
Fixed in [2.0.27](https://github.com/verbb/formie/releases/tag/2.0.27)
